packages/vite-plugin-angular/src/lib/angular-vite-plugin.ts

```diff
@@ -244,8 +257,19 @@ export function angular(
             >('@angular/compiler-cli/linker/babel')
           ).createEs2015LinkerPlugin;
 
+          const forceAsyncTransformation =
```

Comment:
This should be added to the `esbuild` object above

https://github.com/angular/angular-cli/pull/23682/files#diff-a99adf3526ebc926333013ad739eafc4edfc01af13f25822552c49295dea6defR310